translations/fr.json

```diff
@@ -0,0 +1,224 @@
+{
+    "translation": {
+        "🔵 Average": "🔵 Commun",
+        "🟣 Flea Banned": "🟣 Banni du marché aux puces",
+        "🟢 Great": "🟢 Peu Commun",
+        "⭐ Legendary ⭐": "⭐ Légendaire ⭐",
+        "🔴 Poor": "🔴 Mauvais",
+        "🛒 {{trader.name}} restock in {{numMinutes}} minutes 🛒": "🛒 {{trader.name}} se réapprovisionne dans {{numMinutes}} minute(s) 🛒",
+        "❌ ERROR ❌": "❌ ERREUR ❌",
+        "{{thingName}} set to {{level}}.": "IDK",
+        "A Dmg": "IDK",
+        "About": "À propos",
+        "additional results not shown.": "Les résultats additionels ne sont pas montrés.",
+        "All hideout stations set to {{level}}.": "IDK",
+        "All traders set to {{level}}.": "IDK",
+        "all traders": "tous les marchands",
+        "Ammo Table": "Tableau de munitions",
+        "An error has occurred": "Une erreur s'est produite",
+        "Available Commands": "Commandes disponibles",
+        "Barter": "Troquer",
+        "Bosses": "IDK",
+        "Bugs? Missing features? Questions? Chat with us on Discord!": "Des bugs? Des fonctionnalités manquantes? Des questions? Discutez avec nous sur Discord !",
+        "Chance": "Chance",
+        "Changes provided by": "Changements apportés par",
+        "[Click here]({{url}}) for full notes.": "[Cliquez ici]({{url}}) pour les notes complètes.",
+        "Click the link above to invite the Stash bot to your Discord server!": "Cliquez sur le lien au-dessus pour inviter Stash Bot sur votre serveur Discord !",
+        "Come visit us in our server.": "Venez nous voir sur notre serveur.",
+        "Contributors": "Contributeurs",
+        "Craft": "Fabriquer",
+        "Cures": "Remèdes",
+        "Default progress": "IDK",
+        "Delay": "Délai",
+        "Dmg": "IDK",
+        "Duration": "IDK",
+        "EFT Discord Bot": "Bot Discord d'EFT",
+        "Enjoy": "Amusez-vous bien",
+        "Escape from Tarkov Status": "Statut d'Escape from Tarkov",
+        "Escort": "Escorter",
+        "Examples": "Exemples",
+        "Fee": "Frais",
+        "Flea Market": "Marché aux puces",
+        "Flea Price (avg)": "IDK",
+        "Flea Price (low)": "IDK",
+        "Flea Price (optimal)": "IDK",
+        "Found no results for \"{{searchString}}\"": "Pas de résultats pour la recherche \"{{searchString}}\"",
+        "for": "pour",
+        "Format HH:MM:SS": "Format HH:MM:SS",
+        "Frag": "IDK",
+        "Get the full data from {{url}}": "Obtenez les données complètes à partir de {{url}}",
+        "Good Luck!": "Bonne Chance !",
+        "Health": "Vie",
+        "Help for /": "IDK",
+        "Hideout": "Refuge",
+        "Hideout Management skill set to {{managementLevel}}.": "IDK",
+        "I have been online for": "Je suis en ligne depuis",
+        "Intelligence Center": "IDK",
```

Comment:
Intelligence center is a "hideout module". Think of it like a mini CIA/MI6 station 😂